offer-manager-app/src/main/resources/offer-template/offer.html

```diff
@@ -532,6 +532,14 @@ <h4>Project Management & Data Storage (PM & DS)</h4>
                             888.00
                         </div>
                     </div>
+                    <div class="row sub-total-costs" id="DATA_MANAGEMENT-sub-overhead">
+                        <div class="col-10 cost-summary-field">
+                            Project management and data storage:
+                        </div>
+                        <div class="col-2 price-value" id="DATA_MANAGEMENT-overhead-costs-value">
+                            666.00
+                        </div>
+                    </div>
                     <div class="row total-costs single-overscore" id="offer-overhead">
```

Comment:
missing PROJECT_MANAGEMENT here

Reply:
ProjectManagment Items are included in Productgroup.DATA_MANAGEMENT since they are grouped together in the final listing as of now.

Reply:
same suggestion as above, to make the group explicit then